tests/fsharpqa/Source/Conformance/LexicalAnalysis/ConditionalCompilation/ExtendedIfGrammar.fs

```diff
@@ -0,0 +1,182 @@
+// #Conformance #LexicalAnalysis
+
+let success = 1
```

Comment:
These tests might seem a bit unnecessary as the unit tests cover these and more, however it's good IMO to have tests on the same level as the user are operating (ie fsi/fsc)